src/model/selection.js

```diff
@@ -126,6 +126,10 @@ export default class Selection {
 	 * or backward - from end to start (`true`). Defaults to `false`.
 	 */
 	addRange( range, isBackward ) {
+		if ( !( range instanceof Range ) ) {
+			throw new CKEditorError( 'model-selection-invalid-range: Invalid Range.' );
```

Comment:
`model-selection-invalid-range` is inconsistent with other error "codes" that we have in this file, which does not have `model-` prefix.

I just noticed that there are multiple error messages in `view` namespace that have `view-` prefix, but some do not. I see that we have consistency problem regarding this, so I will make follow-up issue, but for now, please make it consistent in one file.